PCbuild/readme.txt

```diff
@@ -52,6 +52,42 @@ Release
     settings, though without PGO.
 
 
+Building Python using Clang/LLVM
+--------------------------------
+
+See https://learn.microsoft.com/en-us/cpp/build/clang-support-msbuild?view=msvc-170
+for how to install and use clang-cl bundled with Microsoft Visual Studio.
+You can use the IDE to switch to clang-cl for local development,
+but because this alters the *.vcxproj files, the recommended way is
+to use build.bat:
+
+build.bat "/p:PlatformToolset=ClangCL" "/p:PreferredToolArchitecture=x64"
+
+All other build.bat options continue to work as with MSVC, so this
+will create a 64bit release binary. PreferredToolArchitecture is needed,
+because msbuild by default selects the 32bit architecture of the toolset,
+which uses -m32 as the default target architecture.
+
+You can also use a specific version of clang-cl downloaded from
+https://github.com/llvm/llvm-project/releases, e.g.
+clang+llvm-18.1.8-x86_64-pc-windows-msvc.tar.xz.
+Given you have extracted that to <my-clang-dir>, you can use it like so
+build.bat --pgo "/p:PlatformToolset=ClangCL" "/p:LLVMInstallDir=<my-clang-dir> "/p:LLVMToolsVersion=18"
+
+Here, PreferredToolArchitecture is not needed, because this is a 64bit
+platform toolset, but LLVMToolsVersion has to be set accordingly.
+Setting the major version is enough, although you can be specific
+and use 18.1.8 in the above example, too.
+
+Even --pgo works out of the box. However, if you want to run the PGO task
```

Comment:
```suggestion
Use the --pgo option to build with instrumented profiling. However, if you want to run the PGO task
```

Let's not sound too surprised that it works 😉 Also would be good to replace "instrumented profiling" with whatever word will help someone who knows Clang already understand exactly what we're using.

Reply:
If the term is the same, that's fine. Main thing is to use this sentence to bridge any terminology gaps, so if they don't exist, then let's just be direct.